tests/containers/isolation.pm

```diff
@@ -43,32 +43,44 @@ sub run {
     select_serial_terminal;
 
     $runtime = $self->containers_factory($args->{runtime});
-    install_packages('jq');
+    my @packages = qw(jq);
+    # rootless docker is not available on SLEM
+    if ($args->{runtime} eq "docker" && !is_transactional) {
+        my $docker_package = check_var("CONTAINERS_DOCKER_FLAVOUR", "stable") ? "docker-stable-rootless-extras" : "docker-rootless-extras";
+        push @packages, qw($docker_package);
+    }
+    install_packages(@packages);
 
     my %ip_addr;
     for my $ip_version (4, 6) {
         my $iface = script_output "ip -$ip_version --json route list match default | jq -r '.[0].dev'";
         $ip_addr{$ip_version} = script_output "ip -$ip_version --json addr show $iface | jq -r '.[0].addr_info[0].local'";
     }
 
-    assert_script_run "$runtime network create --ipv6 --internal $network";
+    my $ipv6_opts = check_var("CONTAINERS_DOCKER_FLAVOUR", "stable") ? "--subnet 2001:db8::/64" : "";
```

Comment:
Please add a comment why this is needed. Or better a link to the reason

Reply:
Fixed.